introduction_to_amazon_algorithms/jumpStart_tabular_classification/Amazon_JumpStart_Tabular_Classification_LightGBM_CatBoost.ipynb

```diff
@@ -0,0 +1,596 @@
+{
```

Comment:
<p>"with the maximum probability over others." seems like it's formatting incorrectly. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
yes, this has been resolved by the revised PR.

Reply:
<p>Yes, I can see it is resolved in other notebooks, but not this one. If this one notebook is corrected, it should be good.</p>

Reply:
Thanks for catching this. I just resolved it in the latest revision.